spec/domain/etl/ga/concerns/transform_path_spec.rb

```diff
@@ -23,27 +23,61 @@ class Dummy
         :ga_event,
         :with_views,
         page_path: "/https://www.gov.uk/topics",
+        bounces: 1,
         upviews: 1,
         pviews: 1,
         useful_no: 1,
         useful_yes: 1,
         searches: 1,
         entrances: 1,
-        exits: 1
+        exits: 1,
+        page_time: 1
       )
     end
 
     before(:each) do
       create(:ga_event,
         :with_views,
         page_path: "/topics",
+        bounces: 1000,
         upviews: 100,
         pviews: 200,
         useful_no: 300,
         useful_yes: 400,
         searches: 500,
         entrances: 600,
-        exits: 700)
+        exits: 700,
+        page_time: 1000)
+    end
+
+    it 'returns the correct number of metrics' do
+      event_attributes = event2.attribute_names
+        .reject { |name| name == 'id' }
+        .reject { |name| name == 'bounces' }
+        .reject { |name| name == 'date' }
+        .reject { |name| name == 'page_path' }
+        .reject { |name| name == 'updated_at' }
+        .reject { |name| name == 'created_at' }
+        .reject { |name| name == 'process_name' }
+        .reject { |name| name == 'page_time' }
```

Comment:
Maybe this could be rewritten, something like?
```ruby
names_to_exclude = %w[id bounces date ...]
event_attributes = event2.attribute_names
  .reject { |name| names_to_exclude.include? name }
```

Reply:
Keep forgetting that's the better way to do things like this. Fixed in https://github.com/alphagov/content-performance-manager/pull/1197/commits/5646194a2d6e75947d5f83716199581850d887af - thanks for the feedback.